contracts/scripts/foundry/InitializeL1ScrollOwner.s.sol

```diff
@@ -165,6 +168,11 @@ contract InitializeL1ScrollOwner is Script {
         _selectors[0] = ScrollMessengerBase.setPause.selector;
         owner.updateAccess(L1_SCROLL_MESSENGER_PROXY_ADDR, _selectors, SCROLL_MULTISIG_NO_DELAY_ROLE, true);
 
+        // no delay, emergency multisig
+        _selectors = new bytes4[](1);
+        _selectors[0] = ScrollMessengerBase.setPause.selector;
```

Comment:
What about `ScrollChain`?

Reply:
Only `L1ScrollMessenger` is related to funds. If private key leaks happen in the prover of sequencer, we may still need to pause `L1ScrollMessenger` to prevent possible future fund loss. So I think using current logic for `ScrollChain` is enough.

Reply:
Let's add `ScrollChain` too because the `ScrollChain` contract verifies the withdraw trie root that allows users to withdraw.